khal/cli.py

```diff
@@ -682,5 +690,18 @@ def configure(ctx):
 
     return cli, interactive_cli
 
+def find_load_plugins():
+    """Find and load all plugins in the plugin directory."""
+    # check all folders in $XDG_DATA_HOME/khal/plugins if they contain a
+    # __init__.py file. If so, import them.
+    plugin_dir = os.path.join(xdg.BaseDirectory.xdg_data_home, 'khal', 'plugins')
+    if not os.path.isdir(plugin_dir):
+        return
+    sys.path.append(plugin_dir)
+    for plugin in os.listdir(plugin_dir):
+        if os.path.isfile(os.path.join(plugin_dir, plugin, '__init__.py')):
+            logger.debug(f'loading plugin {plugin}')
+            __import__(plugin)
+
```

Comment:
Nice and simple approach :+1:

Reply:
@tgy you are right, I'm mixing several things in this PR, they only have in common that they are related to colors. I will split this into two PRs:
1. overriding (elements of) the color theme, will also make more elements themeable and slightly tweak the existing color themes. This one should be good to merge.
2. Plugin support for writing color theme plugins, this one is probably not ready yet.